webpack/scenes/ContentViews/Details/Filters/CVErratumIDFilterContent.js

```diff
@@ -0,0 +1,244 @@
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import PropTypes from 'prop-types';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { TableVariant } from '@patternfly/react-table';
+import { Tabs, Tab, TabTitleText, Split, SplitItem, Button, Dropdown, DropdownItem, KebabToggle } from '@patternfly/react-core';
+import { STATUS } from 'foremanReact/constants';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import onSelect from '../../../../components/Table/helpers';
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import {
+  selectCVFilterErratumID,
+  selectCVFilterErratumIDStatus,
+  selectCVFilterErratumIDError,
+  selectCVFilters, selectCVFilterDetails, selectCVFiltersStatus,
+} from '../ContentViewDetailSelectors';
+import getContentViewDetails, {
+  addCVFilterRule, removeCVFilterRule, getCVFilterErrata,
+  deleteContentViewFilterRules, addContentViewFilterRules,
+} from '../ContentViewDetailActions';
+import AddedStatusLabel from '../../../../components/AddedStatusLabel';
+import ErratumTypeLabel from '../../../../components/ErratumTypeLabel';
+import AffectedRepositoryTable from './AffectedRepositories/AffectedRepositoryTable';
+
+const CVErratumIDFilterContent = ({
+  cvId, filterId, showAffectedRepos, setShowAffectedRepos,
+}) => {
+  const dispatch = useDispatch();
+  const { results: filterResults } =
+    useSelector(state => selectCVFilters(state, cvId), shallowEqual);
+  const response = useSelector(state =>
+    selectCVFilterErratumID(state, cvId, filterId), shallowEqual);
+  const status = useSelector(state =>
+    selectCVFilterErratumIDStatus(state, cvId, filterId), shallowEqual);
+  const filterLoad = useSelector(state =>
+    selectCVFiltersStatus(state, cvId), shallowEqual);
+  const error = useSelector(state =>
+    selectCVFilterErratumIDError(state, cvId, filterId), shallowEqual);
+  const filterDetails = useSelector(state =>
+    selectCVFilterDetails(state, cvId, filterId), shallowEqual);
+  const { repositories = [] } = filterDetails;
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [activeTabKey, setActiveTabKey] = useState(0);
+  const filterLoaded = filterLoad === 'RESOLVED';
+  const loading = status === STATUS.PENDING;
+  const [bulkActionOpen, setBulkActionOpen] = useState(false);
+  const deselectAll = () => setRows(rows.map(row => ({ ...row, selected: false })));
+  const toggleBulkAction = () => setBulkActionOpen(prevState => !prevState);
+  const hasAddedSelected = rows.some(({ selected, added }) => selected && added);
+  const hasNotAddedSelected = rows.some(({ selected, added }) => selected && !added);
+  const columnHeaders = [
+    __('Errata ID'),
+    __('Type'),
+    __('Issued'),
+    __('Updated'),
+    __('Severity'),
+    __('Synopsis'),
+    __('Status'),
+  ];
+
+  /* eslint-disable camelcase */
```

Comment:
Instead of diabling this, use something like errata_id: erratumId or something unique which doesn't cause conflicts..